R/process_extract_delayed_discharges.R

```diff
@@ -22,6 +22,10 @@ process_extract_delayed_discharges <- function(data,
   # Check that the supplied year is in the correct format
   year <- check_year_format(year)
 
+  # Specify years available for running
+  if (year < 1617) {
+    return(NULL)
+  }
```

Comment:
Shouldn't this be checking `data` rather than `year`.

```suggestion
  if (is.null(data)) {
    return(NULL)
  }
```

Reply:
I think this should be year here as the DD extract using `read_extract_delayed_discharges` will pass through data for all years and so we want to filter on this?

Reply:
Yep, you're right!